terraform/services/api-waf/main.tf

```diff
@@ -41,6 +42,25 @@ resource "aws_wafv2_ip_set" "api_customers" {
   }
 }
 
+resource "aws_wafv2_ip_set" "ipv6_api_customers" {
+  count              = var.app == "bcda" ? 1 : 0
+  name               = "${var.app}-${var.env}-ipv6-api-customers"
+  description        = "IP ranges for customers of this API"
+  scope              = "REGIONAL"
+  ip_address_version = "IPV6"
+
+  # Addresses will be managed outside of terraform. This is
+  # a placeholder address for all apps/environments.
+  # See: https://confluence.cms.gov/x/UDs2Q
+  addresses = []
```

Comment:
Can this be applied with an empty array? Possibly! I'm seeing now that [the docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_ip_set) say "Contains an array of strings that specifies zero or more IP addresses or blocks of IP addresses."

If so, would you remove the "This is a placeholder address for all apps/environments." sentence from the comment? And update the addresses attribute in api_customers to match?